core/example/solvers/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(SPME particles.cpp direct.cpp ewald.cpp pme.cpp main.cpp)
+find_package(FFTW3 REQUIRED)
+target_link_libraries(SPME cabanacore ${FFTW3_LIBRARIES})
```

Comment:
```suggestion
target_link_libraries(SPME cabanacore ${FFTW3_LIBRARIES})
target_include_directories(SPME ${FFTW3_INCLUDE_DIRS})
```

Reply:
@sfogerty can you move all the line with `SPME` into the `if(FFTW3_FOUND)` block and remove the duplicated line?